custom/icds_reports/reports/fact_sheets.py

```diff
@@ -12,6 +11,40 @@
 from custom.icds_reports.sqldata.agg_child_health_monthly import AggChildHealthMonthlyDataSource
 from custom.icds_reports.sqldata.national_aggregation import NationalAggregationDataSource
 from custom.icds_reports.utils import person_is_beneficiary_column, default_age_interval
+from custom.icds_reports.cache import icds_quickcache
+
+
+FACT_SHEET_DATASOURCES_NAMES = {
+            'AggChildHealthMonthlyDataSource': AggChildHealthMonthlyDataSource,
+            'AggCCSRecordMonthlyDataSource': AggCCSRecordMonthlyDataSource,
+            'AggAWCMonthlyDataSource': AggAWCMonthlyDataSource
+}
+
+
+@icds_quickcache(['domain', 'previous_month', 'data_source_name', 'loc_level', 'show_test', 'beta'],
+                 timeout=30 * 60)
+def get_data_for_national_aggregatation(domain, previous_month, data_source_name, loc_level, show_test, beta):
+    national_config = {
+        'domain': domain,
+        'previous_month': parse(previous_month),
+    }
+    datasource_argument = {
+        'config': national_config,
+        'loc_level': loc_level,
+        'show_test': show_test,
+    }
+
+    if data_source_name != 'AggCCSRecordMonthlyDataSource':
+        datasource_argument['beta'] = beta
```

Comment:
not sure this is actionable, but it seems a little weird that the beta argument is not passed through to this report. would it make sense to add it as an (unused) argument? or that feels silly?